### PR TITLE
Program:GCI

### PR DESCRIPTION
**Pull request by devmrfitz**
Screenshot post correction
Errors reprted by pep8

### CODE REVIEW

**Review by namrata-jha — APPROVED** [review]
The code looks fine, thanks Fitz!

**Review by namrata-jha — APPROVED** [review]

File: app/api/dao/task.py [hunk 1]
```diff
@@ -49,24 +56,29 @@ def create_task(user_id, mentorship_relation_id, data):
...
+            The last element is the HTTP response code
         """
 
         user = UserModel.find_by_id(user_id)
         relation = MentorshipRelationModel.find_by_id(mentorship_relation_id)
         if relation is None:
             return messages.MENTORSHIP_RELATION_DOES_NOT_EXIST, 404
 
-        if not (user_id == relation.mentee_id or user_id == relation.mentor_id):
+        if user_id != relation.mentee_id and user_id != relation.mentor_id:
```
Comment (Tlazypanda) [inline app/api/dao/task.py:81]: Hey @devmrfitz , can you explain these changes?

**Review by Tlazypanda — COMMENTED** [review]
Hey @devmrfitz , Great work!  Just left a comment regarding a change if you could just look into that since I don't think pep8 was throwing a warning for the change.
Must say blazing fast work :tada:

File: app/api/dao/task.py (hunk 1 above)
Comment (devmrfitz) [reply app/api/dao/task.py:81]: I converted (not(A or B)) into ((not A) and (notB)) 
This shortened the length of the line as directed by pycodestyle/pep8

**Comment by Tlazypanda** [comment]
Hey @devmrfitz , can you just reduce the length of line by dividing it into the next line without change in not expression since it reduces code readability ?

**Comment by devmrfitz** [comment]
Thanks for the suggestion @Tlazypanda . Should I follow this in future commits or should I modify the current pull request too?

**Comment by Tlazypanda** [comment]
@devmrfitz, just noticed that base branch of the pr is develop instead of gci-dev.Can you make a new pr from the gci-dev branch as base branch and with the changes requested?

**Comment by isabelcosta** [comment]
I will close since this task work was already merged.
Thank you for your contribution @devmrfitz

**Pull request closed without merging**